app/src/main/java/org/kiwix/kiwixmobile/zimManager/ZimManageViewModel.kt

```diff
@@ -16,722 +16,723 @@
  *
  */
 
-package org.kiwix.kiwixmobile.zimManager
```

Comment:
Why are there unnecessary changes in the file? Please remove unnecessary white spaces.

![Screenshot from 2025-05-27 12-43-18](https://github.com/user-attachments/assets/b0be4b9a-99e4-4df7-8a9f-4789d8d077d7)